src/main/java/com/xatkit/core/recognition/processor/PerspectiveapiInterface.java

```diff
@@ -0,0 +1,281 @@
+                AttributeType.SEXUALLY_EXPLICIT,
+                AttributeType.FLIRTATION,
+                AttributeType.ATTACK_ON_AUTHOR,
+                AttributeType.ATTACK_ON_COMMENTER,
+                AttributeType.INCOHERENT,
+                AttributeType.INFLAMMATORY,
+                AttributeType.LIKELY_TO_REJECT,
+                AttributeType.OBSCENE,
+                AttributeType.SPAM,
+                AttributeType.UNSUBSTANTIAL
+        });
+        languageAttributes.put("es", new AttributeType[] {
+                AttributeType.TOXICITY,
+                AttributeType.SEVERE_TOXICITY,
+                AttributeType.IDENTITY_ATTACK_EXPERIMENTAL,
+                AttributeType.INSULT_EXPERIMENTAL,
+                AttributeType.PROFANITY_EXPERIMENTAL,
+                AttributeType.THREAT_EXPERIMENTAL
+        });
+        this.setRequestedAttributes();
+    }
+
+    /**
+     * Sets the attributes available for the chosen language
+     */
+    private AttributeType[] setRequestedAttributes() {
+        if (language.equals("es")) {
+            requestedAttributes = languageAttributes.get("es");
+        }
+        else {
+            requestedAttributes = languageAttributes.get("en");
+        }
+        return requestedAttributes;
+    }
+
+    /**
+     * Sets the comment that will be analyzed
+     */
+    private String setCommentText(String commentText) {
+        this.commentText = commentText;
+        return commentText;
+    }
+
+    /**
+     * Analyzes the comment and returns the punctuation for each attribute
+     *
+     * @param comment the comment
+     * @return the hash map with attributes and their respective scores
+     */
+    public HashMap<String, Double> analyzeRequest(String comment) {
+        JSONObject response = new JSONObject();
+        HashMap<String, Double> scores = new HashMap<>();
+        try {
+            for (AttributeType k : AttributeType.values()) {
+                scores.put(attributesPrefix + k.toString(), -1.);
+            }
+            this.setCommentText(comment);
+            request = this.createJSONObjectRequest();
+            response = Unirest.post("https://commentanalyzer.googleapis"
+                    + ".com/v1alpha1/comments:analyze?key=" + apiKey)
+                    .header("Content-Type", "application/json")
+                    .body(request)
+                    .asJson().getBody().getObject();
+
+            JSONObject attributes = response.getJSONObject("attributeScores");
+            Double score;
+            for (String k : attributes.keySet()) {
+                score = attributes.getJSONObject(k).getJSONObject("summaryScore").getDouble("value");
+                scores.put(attributesPrefix + k, score);
+            }
+        } catch (Exception e) {
+            Log.warn("PerspectiveAPI failed to compute toxicity scores");
+            System.out.println("PerspectiveAPIResponse=" + response.toString());
+            e.printStackTrace();
```

Comment:
Same as `System.out.println`, don't use `printStackTrace`, excepted when you debug stuff of course